typedspark/_core/dataset.py

```diff
@@ -86,6 +86,11 @@ def _add_schema_metadata(self) -> None:
         for field in self._schema_annotations.get_structtype().fields:
             self.schema[field.name].metadata = field.metadata
 
+    @property
+    def typedspark_schema(self) -> Schema:
```

Comment:
Kunnen we het return-type hier `T` maken? From `DataSet[T]`.

Reply:
Yes!

Reply:
The typechecker doesnt like this..

Reply:
Just leave a `#type: ignore` at the return statement

Reply:
Taking a second look, shouldnt it be `Type[T]`?

Reply:
Ah, yeah, you're right!

Reply:
So its wrong in other places aswell?